fr/integrations/plugin-packs/procedures/applications-monitoring-centreon-mbi.md

```diff
@@ -3,122 +3,329 @@ id: applications-monitoring-centreon-mbi
+### Ojets supervisés
 
-``` shell
-yum install centreon-nrpe-plugin centreon-plugin-Operatingsystems-Linux-Snmp
+* Serveur de reporting Centreon MBI
+
+### Métriques collectées
+
+En plus des modes et des métriques détaillés ci-après, il est également possible
+de superviser les éléments suivants: 
+
+* DWH-db-content : Vérifier si les données présentes dataware house sont à jour
+* DWH-partitions : Vérifier si toutes les partitions MySQL partitions sont à jour
+* failed-jobs : Vérifier si des tâches ont échouées
+* Ntp : Vérifier le décalage de temps du serveur avec le serveur ntp
+* proc-cbis : Status du processus cbis sur le serveur de reporting
+* proc-mysql : Status du processus MySQL sur le serveur de reporting
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--cpu-->
+
+| Metric name                        | Description                                   |
+| :--------------------------------- | :-------------------------------------------- |
+| cpu.utilization.percentage         | CPU utilization. Unit : %                     |
+| core.cpu.utilization.percentage    | Per Core CPU utilization. Unit : %            |
+
+<!--Memory-->
+
+| Metric name             | Description                                              |
+| :---------------------  | :------------------------------------------------------- |
+| memory.usage.bytes      | Memory usage on the device. Unit : Bytes                 |
+| memory.free.bytes       | Free memory on the device. Unit : Bytes                  |
+| memory.usage.percentage | Percentage of Memory usage on the device. Unit : %       |
+| memory.buffer.bytes     | Buffered Memory allocation. Unit : Bytes                 |
+| memory.cached.bytes     | Cached Memory allocation. Unit : Bytes                   |
+| memory.shared.bytes     | Shared Memory allocation. Unit : Bytes                   |
+
+<!--Swap-->
+
+| Metric name                 | Description                                       |
+| :-------------------------- | :------------------------------------------------ |
+| swap.usage.bytes            | Used swap. Unit: Bytes                            |
+| swap.free.bytes             | Free swap. Unit: Bytes                            |
+| swap.usage.percentage       | Percentage of used swap. Unit: %                  |
+
+<!--Load-->
+
+| Metric name                 | Description                                        |
+| :-------------------------- | :------------------------------------------------- |
+| load1                       | System load 1 minute-sample                        |
+| load5                       | System load 5 minutes-sample                       |
+| load15                      | System load 15 minutes-sample                      |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prérequis
+
+Pour superviser un serveur de reporting Centreon MBI, les services NRPE3 et SNMP
+doivent être installés et configurés.
+
+### NRPE3
```

Comment:
Maybe add link to send the user on the Linux NRPE3 Monitoring Proc and add just the necessary configuration:
```
command[check_partitions]=/usr/share/centreon-bi/etl/centreonbiMonitoring.pl --partitions
command[check_db]=/usr/share/centreon-bi/etl/centreonbiMonitoring.pl --db-content
command[check_jobs]=/usr/share/centreon-bi/etl/centreonbiMonitoring.pl --jobs
```